includes/functions.php

```diff
@@ -4119,3 +4119,22 @@ function dokan_get_withdraw_threshold( $user_id ) {
 
     return ( $threshold_day ) ? absint( $threshold_day ) : 0;
 }
+
+/**
+ * Dokan Withdraw Limit Value Validation
+ *
+ * @param mixed $option_value
+ * @param mixed $option_name
+ *
+ * @since DOKAN_LITE_SINCE
+ *
+ * @return mixed $optins
+ */
+function dokan_withdraw_limit_value_validation( $option_value, $option_name ) {
+    if( $option_value['withdraw_limit'] < 0 ) {
+        $option_value['withdraw_limit'] = 0;
```

Comment:
This should be re-written proper way. and need to fix formatting. Do you exactly know, you could get `'withdraw_limit'` there. Cause, it's a global filter hook for settings every option's save.

This could be re-written as below - 
```php
if ( ! empty( $option_value['withdraw_limit'] ) ) {
    $option_value['withdraw_limit'] = $option_value['withdraw_limit'] < 0 ? 0 : $option_value['withdraw_limit'];
}
```